.goreleaser.yml

```diff
@@ -36,7 +36,7 @@ dockers:
       - "go.mod"
       - "go.sum"
       - "main.go"
-      - "proto"
+      - "proto/v1alpha1"
```

Comment:
the new mocks will live in `proto/v1alpha1fakes`, which I don't want to include in the docker image